libs/can/api.h

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2021 Olin Electric Motorsports
+ */
+#pragma once
+#include <stdbool.h>
+#include <stdint.h>
+
+typedef enum {
+    BAUD_250KBPS,
+    BAUD_500KBPS,
+} baud_rate_t;
+
+typedef struct {
+    uint16_t id;
+    uint8_t dlc;
```

Comment:
What's a CAN dlc?

Reply:
Nvm this was very googleable. In case anyone else is curious it's the data length code indicating the number of data bytes in the message.